devtools/run-tests.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+set -ev
+
+BACKEND_CHANGE=`git diff ../src/backend`
```

Comment:
you're right, in fact missing a git diff <<master>> ../src/backend, but your command is better